hedera-mirror-rest/config.js

```diff
@@ -173,6 +173,19 @@ const parseMaxTimestampRange = () => {
   conf.maxTimestampRangeNs = BigInt(ms) * 1000000n;
 };
 
+const parseNetworkConfig = () => {
+  const networkConf = getConfig().network;
+  if (networkConf.currencyFormat) {
+    const currencyFormat = networkConf.currencyFormat;
```

Comment:
Use null safe traversal operator in case no network stanza. Can also condense

```suggestion
  const currencyFormat = getConfig()?.network?.currencyFormat;
  if (currencyFormat) {
```

Reply:
I had not been aware of that "null-safe traversal operator" -- very nice!

Reply:
The code has been condensed, as requested.